### DISCUSSION TITLE
[BUG]No longer working after presumed API change

### DISCUSSION THREAD

**Discussion by ullix** [python-telegram-bot/python-telegram-bot#2718] [Q&A]
### Steps to Reproduce

1. create a bot => ok
2. set up with: myTelegramBot = telegram.Bot(token=myTelegramToken) => ok
3. call: updates = myTelegramBot.get_updates() => 0k
4. get chat id: myTelegramChatID = updates[0].message.from_user.id  => **FAILS**

The failure exception message is: "'NoneType' object has no attribute 'from_user''"

I had it running successfully for a few days, but then it failed suddenly. "updates" is now a list of 15 items, and updates[0] does not even contain "message", and neither of the 15 items contains "from_user".

I don't recall any Telegram updates in the last day, but I now have "Telegram Desktop version 3.1" running on Ubuntu Linux 16.04.

### Expected behaviour

_

### Actual behaviour

_

### Operating System

Ubuntu Linux 16.04

### Version of Python, python-telegram-bot & dependencies

```shell
$ python -m telegram
python-telegram-bot 13.7
Bot API 5.3
certifi 2020.12.05
Python 3.9.4 (default, Apr  9 2021, 01:15:05)  [GCC 5.4.0 20160609]
```

**Comment by Poolitzer** [comment]
Hello there.

No, there weren't API updates, neither are existing workflows breaking. `.message` is just an optional attribute of the `Update` class and there are several non message updates, so your code needs to deal with them

**Comment by ullix** [comment]
Well, I am left a bit puzzled. I am working with this python-telegram-bot only for a few days, and like its simplicity. But as far as can see the "chat_id" is stringently required for "mybot.send_message()". 

But this chat_id may or may not be available in a call, depending on unknown circumstances. Is there any method which reliably returns this chat_id? Or is there a send command, which does not need it?

**Comment by Poolitzer** [comment]
> But this chat_id may or may not be available in a call, depending on **unknown circumstances**. 

Well thats not true, you can be pretty sure what type of updates your bot faces depending on the settings you did to the bot in botfather.

> Is there any method which reliably returns this chat_id?

Well, chat id is not always the same as the from_user id. But [update.effective_chat](https://python-telegram-bot.readthedocs.io/en/stable/telegram.update.html#telegram.Update.effective_chat) will be there for every update which has a chat attribute, so you can use that.

> Or is there a send command, which does not need it?

No, telegram needs to know where to send your message to after all. It always needs a chat_id.

**Comment by ullix** [comment]
Trying `update.effective_chat ` with whatever ID I could find in `updates` always resulted in `None`.

Using `myTelegramChatID = updates[1].message.chat.id` gave me an ID, which, when used in 'send_message' resulted in an Exception:
`telegram.error.Unauthorized: Forbidden: bot was kicked from the group chat`

Not very promising, I'm afraid.

**Reply by Poolitzer** [reply]
... well the bot was kicked from the group. Im not sure what the issue is?
